manifest.json

```diff
@@ -267,6 +267,29 @@
               "fedora-32-helix-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
             }
           }]
+        },
+        {
+          "platforms": [{
+            "dockerfile": "src/fedora/33/amd64",
+            "os": "linux",
+            "osVersion": "fedora33",
+            "tags": {
+              "fedora-33-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+              "fedora-33": {
+                "isLocal": true
+              }
+            }
+          }]
+        },        
```

Comment:
I am probably missing something. But, why do we need a non-Helix queue for Fedora/33❔

Reply:
> I am probably missing something. But, why do we need a non-Helix queue for Fedora/33❔

It's not a queue, it's a base image;  while I don't fully know who the user was for fedora 32, this pattern was there and allows building other .NET Core ready images off Fedora 32 and reduces the amount of stuff Helix images install to primarily helix-specific images.